app/models/driver.rb

```diff
@@ -0,0 +1,16 @@
+class Driver < ApplicationRecord
+  has_many :trips, dependent: :destroy
+  validates :name, presence: true
+  validates :vin, presence: true, uniqueness: true, length: {is: 17}
+
+  def average_rating
+    return trips.sum(&:rating).to_f/ trips.length
+  end
```

Comment:
What happens if there are no trips associated with a driver? Then total would be 0.0 and trip_count is 0. Ruby evaluates 0.0 / 0 as NaN or "Not A Number." When it gets to the driver/show.html.erb view, NaN.to_s evaluates to just "NaN" so it shows up on the driver page as "NaN" on the page.